tests/E2E Tests/CustomSignedAssertionProviderTests/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "AzureAd": {
+        "Instance": "https://login.microsoftonline.com/",
+        "TenantId": "msidlab4.onmicrosoft.com",
+        "ClientId": "f6b698c0-140c-448f-8155-4aa9bf77ceba",
+        "ClientCredentials": [
+            {
+                "SourceType": "CustomSignedAssertion",
+                "CustomSignedAssertionProviderName": "MyCustomExtension"
```

Comment:
I'm confused on this, the docs say this should be the FullName of the type, that should include the namespace, shouldn't it?

Reply:
This lines up with the work item but the code comments contradict the behavior.

Can we fix this comment here: 

https://github.com/AzureAD/microsoft-identity-web/blob/ebc7128489ccdb52b70fcd60c413f4a502ea8e60/src/Microsoft.Identity.Web.Certificate/DefaultCredentialsLoader.CustomSignedAssertion.cs#L41

To match the implementation that it can be provider name or fully qualified type name?